tool/cli.js

```diff
@@ -891,6 +893,61 @@ if (Mozilla && !Mozilla.dntEnabled()) {
     })
   )
 
+  .command(
+    "frontmatter-inventory",
+    "Create frontmatter content inventory as JSON"
+  )
+  .argument("<output>", "Output to file or stdout", {
```

Comment:
I found this pretty surprising, when I tried to use. Typically, if a CLI program takes an argument for writing to a file, then the argument is the path to the output file (e.g., `yarn tool inventory output.json`) and no argument or the special `-` argument writes to stdout (e.g., `yarn tool inventory -` writes to stdout).

But this writes to a hard-coded path. I'm not sure what the benefit of that is. But since the _only_ thing this does so far is spit out some text, I think it'd be better to have no argument at all. If you want to write to a file, then you can use the shell for that (e.g., `yarn tool inventory > ./inventory.json`).

(See https://clig.dev/#arguments-and-flags if, like me, you really like to get into the weeds on this.)